CMakeLists.txt

```diff
@@ -0,0 +1,160 @@
+#
+# CMakeLists.txt
+# Copyright (c) 2019 Serein Pfeiffer <serein.pfeiffer@gmail.com>
+#
+# This file is part of the Box2D QML plugin.
+#
+# This software is provided 'as-is', without any express or implied warranty.
+# In no event will the authors be held liable for any damages arising from
+# the use of this software.
+#
+# Permission is granted to anyone to use this software for any purpose,
+# including commercial applications, and to alter it and redistribute it
+# freely, subject to the following restrictions:
+#
+# 1. The origin of this software must not be misrepresented; you must not
+#    claim that you wrote the original software. If you use this software in
+#    a product, an acknowledgment in the product documentation would be
+#    appreciated but is not required.
+# 2. Altered source versions must be plainly marked as such, and must not be
+#    misrepresented as being the original software.
+# 3. This notice may not be removed or altered from any source distribution.
+#
+cmake_minimum_required (VERSION 3.9 FATAL_ERROR)
+project (qmlbox2d)
```

Comment:
Regarding packaging, I think we should provide 2 different names for the lib depending if we are Qt5 or Qt6, what do you think?

Reply:
That seems strange to me since the plugins for Qt 5 would be expected to be in a different directory than the ones for Qt 6, right? Is there any precedence for including the Qt version in the plugin names?

Reply:
you're right. I just check the Qt plugins and they have the same names. I confused with the library names (Qt5Core, Qt6Core)